backend/src/main/java/cz/cuni/mff/xrg/odcs/backend/execution/pipeline/Executor.java

```diff
@@ -136,21 +135,12 @@
      */
     private String backendID;
 
-    private boolean clusterMode = false;
-
     /**
      * Sort pre/post executors.
      */
     @PostConstruct
     public void init() {
-        try {
-            this.clusterMode = this.appConfig.getBoolean(ConfigProperty.BACKEND_CLUSTER_MODE);
-        } catch (MissingConfigPropertyException e) {
-            LOG.info("Running in single mode because cluster mode property is missing in config.properties, {}", e.getLocalizedMessage());
-        }
-        if (this.clusterMode) {
-            this.backendID = this.appConfig.getString(ConfigProperty.BACKEND_ID);
-        }
+        this.backendID = this.appConfig.getString(ConfigProperty.BACKEND_ID);
```

Comment:
Note for tomas-knap: what happens if the config.properties BACKEND_ID is missing?